spezi-module/contact/src/main/java/edu/stanford/spezikt/spezi_module/contact/component/ContactOptionCard.kt

```diff
@@ -0,0 +1,96 @@
+package edu.stanford.spezikt.spezi_module.contact.component
+
+import android.content.Intent
+import android.net.Uri
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.layout.wrapContentSize
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.Email
+import androidx.compose.material3.Card
+import androidx.compose.material3.Icon
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import edu.stanford.spezikt.core.designsystem.theme.SpeziKtTheme
+import edu.stanford.spezikt.spezi_module.contact.model.ContactOption
+import edu.stanford.spezikt.spezi_module.contact.model.ContactOptionType
+import java.util.UUID
+
+/**
+ * A card that displays a contact option.
+ * @param option The contact option to display.
+ * @sample edu.stanford.spezikt.spezi_module.contact.component.ContactOptionCardPreview
+ * @see ContactOption
+ * @see ContactOptionType
+ * @see edu.stanford.spezikt.spezi_module.contact.ContactScreen
+ */
+@Composable
+fun ContactOptionCard(option: ContactOption) {
+    val context = LocalContext.current
+    Card(
+        modifier = Modifier
+            .wrapContentSize(Alignment.Center)
+            .width(90.dp)
+            .clickable {
+                when (option.optionType) {
+                    ContactOptionType.EMAIL -> {
+                        val emailIntent = Intent(Intent.ACTION_SENDTO).apply {
+                            data = Uri.parse("mailto:${option.value}")
+                        }
+                        context.startActivity(emailIntent)
```

Comment:
This logic should not be handled here. The views should be as dummy as possible 😅 . As proposed in the Bluetooth view model, here we could have the following setup

```kotlin
// extract from all the actions supported by the view model
sealed interface OnAction {
    data class CardClicked(val option: ContactOption): OnAction
}

// new signature
fun ContactOptionCard(option: ContactOption, publisher: (OnAction) -> Unit)

// clickable
.clickable {
   publisher(OnAction.CardClicked(option = option))
}
```